PresentScreenings/Controllers/ScreeningDialogController.cs

```diff
@@ -85,6 +86,10 @@ public override void ViewWillDisappear()
         {
             // Tell the presentor we're gone.
             _presentor.RunningPopupsCount--;
+            if (_presentor.RunningPopupsCount == 0)
+            {
+                _presentor.ScreeningInfoDialog = null;
+            }
```

Comment:
Please consider to prevent technical debt and make this code less dependent from the Screening Info dialog being alway the only active dialog.